subsys/bluetooth/audio/audio_internal.h

```diff
@@ -39,9 +39,14 @@ ssize_t bt_audio_write_chrc(struct bt_conn *conn, const struct bt_gatt_attr *att
 	.user_data = _user_data, \
 }
 
+/* Suppress the function pointer to constant comparison warning */
+#define _IS_NULL(_ptr) ((uintptr_t)(_ptr) == (uintptr_t)NULL)
```

Comment:
Can you elaborate on this? 

What error/warning do you get by comparing a function pointer in `BT_AUDIO_CHRC` with `NULL` directly? What we simply cast it to `void *`?

Reply:
I'll probably remove this one